### ISSUE TITLE
Unsatisfiable requirements detected for package cuDNN

### ISSUE THREAD

**Issue by vonpetersenn** [darsnack/FluxPrune.jl#9]
Hello,

when trying to add `FluxPrune` to my project that includes `Flux`, `CUDA` and `cuDNN`, there are version conficts that cannot be resolved.

I removed the packages `Flux`, `CUDA`, and `cuDNN` to first add `FluxPrune` and let it choose the `Flux` and `CUDA` version that it supports. But then I have a _Unsatifiable requirements_ when trying to add `cuDNN`:

Original error message:

pkg> add FluxPrune
    Updating registry at `~/.julia/registries/General.toml`
   Resolving package versions...
ERROR: Unsatisfiable requirements detected for package Flux [587475ba]:
 Flux [587475ba] log:
 ├─possible versions are: 0.4.1-0.14.15 or uninstalled
 ├─restricted to versions * by an explicit requirement, leaving only versions: 0.4.1-0.14.15
 ├─restricted by compatibility requirements with FluxPrune [1384c1f3] to versions: 0.12.0-0.13.17
 │ └─FluxPrune [1384c1f3] log:
 │   ├─possible versions are: 0.1.0 or uninstalled
 │   └─restricted to versions * by an explicit requirement, leaving only versions: 0.1.0
 ├─restricted by compatibility requirements with CUDA [052768ef] to versions: [0.4.1-0.10.4, 0.13.12-0.14.15] or uninstalled, leaving only versions: 0.13.12-0.13.17
 │ └─CUDA [052768ef] log:
 │   ├─possible versions are: 0.1.0-5.2.0 or uninstalled
 │   ├─restricted to versions * by an explicit requirement, leaving only versions: 0.1.0-5.2.0
 │   ├─restricted by julia compatibility requirements to versions: [2.3.0, 2.5.0-5.2.0] or uninstalled, leaving only versions: [2.3.0, 2.5.0-5.2.0]
 │   ├─restricted by compatibility requirements with cuDNN [02a925ec] to versions: 4.0.0-5.2.0
 │   │ └─cuDNN [02a925ec] log:
 │   │   ├─possible versions are: 1.0.0-1.3.0 or uninstalled
 │   │   ├─restricted to versions * by an explicit requirement, leaving only versions: 1.0.0-1.3.0
 │   │   └─restricted by compatibility requirements with CUDA [052768ef] to versions: 1.0.0-1.1.1 or uninstalled, leaving only versions: 1.0.0-1.1.1
 │   │     └─CUDA [052768ef] log: see above
 │   └─restricted by compatibility requirements with Flux [587475ba] to versions: 2.1.0-4.4.2, leaving only versions: 4.0.0-4.4.2
 │     └─Flux [587475ba] log: see above
 └─restricted by compatibility requirements with Functors [d9f16b24] to versions: [0.4.1-0.10.4, 0.12.0-0.13.4] or uninstalled — no versions left
   └─Functors [d9f16b24] log:
     ├─possible versions are: 0.1.0-0.4.10 or uninstalled
     └─restricted by compatibility requirements with FluxPrune [1384c1f3] to versions: 0.2.8
       └─FluxPrune [1384c1f3] log: see above

**Comment by darsnack** [comment]
Yes this project has seen no updates while Flux and CUDA have had many. The existing Project.toml probably needs up to date compatibility bounds. Or ideally, the package should be updated to the latest Flux.

**Comment by cirobr** [comment]
Hello, same here.
